package.json

```diff
@@ -197,6 +197,21 @@
         "filetype": "ts",
         "tokenCase": "camel"
       },
+      "colors": {
+        "exclude": {
+          "group": [
+            "Internal"
+          ],
+          "token": "\\b(Figma)\\b.*"
+        }
+      },
+      "typography": {
+        "exclude": {
```

Comment:
Here it might be better to use the "include" way to only pick the "Native" typography styles:

```json
{
  "include": {
    "group": ["Native"]
  }
}
```